experiment/resultstore/convert.go

```diff
@@ -124,6 +127,32 @@ func convert(project, details string, url gcs.Path, result downloadResult) (resu
 				URL:         buildLog, // ensure build-log.txt appears as the invocation log
 			},
 		},
+		Properties: []resultstore.Property{
+			{
+				Key:   "Job Name",
```

Comment:
Can we avoid key names with spaces? `job` seems sufficient here

Reply:
Done